.hooks/bootstrap

```diff
@@ -130,3 +134,18 @@ else
     fi
 fi
 
+echo "======================="
+
+if [ -n "$NO_NPM" ]; then
+    echo "Skipping npm node modules setup as environment varialbe NO_NPM is set"
+else
+    if ! command -v npm >/dev/null 2>&1; then
```

Comment:
you can add this code in update flow too, in case user already have hooks and he runs ./hooks/bootstrap -u

Reply:
It's there. At line 38. We just check if the node_modules dir exists as an indication.

Reply:
but if all the developers that didn't install node_modules and have bootstrap installed, when they will run `.hooks/bootstrap -u` , the process will try to find `node_modules` and will not find it.
It is not a big deal, developers can delete the .venv and install the hook again

Reply:
It is better to do accordingly to the folder. So if someone choose to not install we don't do this every time. Devs can run nom install if needed.